Empire/Empire.cpp

```diff
@@ -1017,6 +1017,18 @@ bool Empire::Ready() const
 void Empire::SetReady(bool ready)
 { m_ready = ready; }
```

Comment:
I do not check all the SetReady code paths, but would it possible that a client sends further messages after sending AUTO_TURN ?

I mean like pressing the turn button (or reentering the game) in order to signal that you are not ready anymore/want to change orders. In that case m_auto_turn_count needs to be set to zero i guess.

Reply:
Yes, if player connected and made orders auto-turn should be disabled.

Reply:
> Yes, if player connected and made orders auto-turn should be disabled.

Ok; Probably this should be handled on the server and not the client.
m_auto_turn_count only gets set zero only when getting AUTO_TURN message from client, so I guess that means it is missing

Reply:
a thanks, I see the newest commit should fix this